modules/fundamental/tests/cve.rs

```diff
@@ -0,0 +1,84 @@
+#![allow(clippy::expect_used)]
+
+use anyhow::bail;
+use futures_util::stream;
+use std::convert::Infallible;
+use test_context::test_context;
+use test_log::test;
+use trustify_common::id::Id;
+use trustify_module_ingestor::service::{Format, IngestorService};
+use trustify_test_context::TrustifyContext;
+
+#[test_context(TrustifyContext)]
+#[test(tokio::test)]
+async fn reingest_cve(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let ingestor = IngestorService::new(ctx.graph.clone(), ctx.storage.clone());
+
+    let data = ctx.document_bytes("cve/CVE-2021-32714.json").await?;
```

Comment:
You might simplify this test by using `ctx.ingest_document[s]`. That'll return you the id's you need but saves you from creating your own `IngestorService`.

Reply:
Done.